web/themes/custom/jcc_components/templates/paragraph/paragraph--hero.html.twig

```diff
@@ -40,12 +40,12 @@
 #}
 
 {% set parent_type = paragraph.getParentEntity().getType() %}
-{% set variant = paragraph.field_variant.value|json_decode %}
+{% set no_aside = content.field_subheading.0 is empty and content.field_aside.0 is empty and content.field_link.0 is empty %}
 {% set background = parent_type == 'landing_page' ? 'alt' : 'default' %}
 
 {% include "@molecules/hero/hero.twig" with {
   hero: {
-    layout: variant.hero,
+    layout: no_aside == 1 ? 'full-main' : hero.variant,
```

Comment:
1. It should be `variant.hero`, not `hero.variant`
2. I think you can just do `layout: no_aside ? 'full-main' : variant.hero,`